.tmuxinator.yml

```diff
@@ -10,15 +10,12 @@ pre_window:
   - alias mint_rpc_client="\$FM_MINT_RPC_CLIENT"
 tmux_detached: false
 windows:
-  - main:
-      panes:
-        - user:
-            - source scripts/tmux-user-shell.sh
```

Comment:
Can we refactor the shell script such that we can keep both versions? The intention of having a separate window was that it makes copy/pasting using the terminal emulator easier. Otherwise you'd need to switch the pane you are copying from to full-screen or use tmux's internal copy/paste facilities if they even exist.

Reply:
yup makes sense changed back to what it was before 🚀